com.incquerylabs.v4md/src/main/com/incquerylabs/v4md/internal/ProfilingMagicDrawProjectNavigationHelper.java

```diff
@@ -8,13 +8,41 @@
 import org.eclipse.viatra.query.runtime.base.api.BaseIndexOptions;
 import org.eclipse.viatra.query.runtime.base.core.NavigationHelperContentAdapter;
 
-import com.google.common.base.Stopwatch;
 import com.nomagic.uml2.ext.jmi.EventSupport;
 
 public class ProfilingMagicDrawProjectNavigationHelper extends MagicDrawProjectNavigationHelper {
 
+	public class StopWatch {
```

Comment:
The stopwatch logic is very hard to understand, as the names of the variables are not descriptive enough, making it very challenging to understand the getElapsed() method.

Naming the time variables like `currentStartTime` and `totalSavedTime` would reduce the confusion.